Improve some comments in cert.pl (fixes #749)

kini opened the pull request:
See #749 for discussion. Sorry I took so long to do this.

pennyan commented:
Looks good to me! Think you for cleaning it up! I found one typo! "A third option is tell" -> "A third option is to tell" ?

kini commented:
Good catch! Fixed.

kini commented:
Thanks for the review! I made one more change -- I noticed that I had removed some text about .acl2 files, so I added that back in with some further clarification. To avoid more back-and-forth I'll just merge the PR now, but feel free to back out or edit the new paragraph I added (starting "This script also scans the [...]") if you don't like it.

The reason I put the "incidentally" in at the beginning of the last paragraph is that the previous paragraph, which says "At this point, `cert.pl` should be configured properly and ready to use.", is referring back to (and wrapping up) the entire doc topic, whose purpose is to help the user set up the environment in which they should be running `cert.pl`. In contrast, the last paragraph is about what happens when you don't follow the directions in the doc topic, and let `cert.pl` figure things out for itself; furthermore, it's only relevant to the books directory section and not the whole doc topic. Ideally I would have liked to put it before the "At this point [...]" sentence, but then it's weird because the "if `cert.pl` cannot determine [...]" paragraph is not an instruction to the reader, so one loses the sense of a "timeline of actions" while reading that paragraph, making "At this point" seem incongruous afterwards. So I figured the easiest thing would be to put that out-of-place paragraph at the end but with an "Incidentally," in front, to mark it as a postscript and make it seem like less of a non-sequitur. Oh well, maybe I'm overthinking it, lol.
